docker-compose.yml

```diff
@@ -20,9 +26,9 @@ services:
             - .:/var/www/html
             - ./var/log:/var/log/nginx
     db:
-        image: postgres:9
+        image: postgres:9.6
```

Comment:
I think we can upgrade to 10.x 
@atrutko-4xxi any concerns from your side?

Reply:
@artem328 I agree with @etomilin , you need to start using version 10+
The main limitation on using Postgres version 10 is doctrine/dbal v2.7.0+
https://github.com/doctrine/dbal/releases